osc-server/src/test/java/org/osc/core/broker/service/servicefunctionchain/UpdateServiceFunctionChainServiceTest.java

```diff
@@ -82,14 +84,67 @@ public void testDispatch_WhenVsIdsListUpdateValidationSuccessful_ValidationSucce
 		request.setDto(dto);
 		request.setName(this.sfc.getName());
 		List<Long> vsIds = new ArrayList<Long>();
+		vsIds.add(this.vs.getId());
+		vsIds.add(this.vs1.getId());
+		request.setVirtualSystemIds(vsIds);
+		Mockito.when(this.service.validator.validateAndLoad(request)).thenReturn(this.sfc);
+		// Act.
+		BaseJobResponse response = this.service.dispatch(request);
+		Assert.assertNotNull("The returned response should not be null.", response);
+	}
+	
+	@Test
+	public void testDispatch_sfcVirutalSystemIdsInOrder_ValidationSucceeds() throws Exception {
+		
+		BaseDto dto = new BaseDto();
+		dto.setId(this.sfc.getId());
+		dto.setParentId(this.vc.getId());
+		request.setDto(dto);
+		request.setName(this.sfc.getName());
+		List<Long> vsIds = new ArrayList<Long>();
 		vsIds.add(this.vs1.getId());
 		vsIds.add(this.vs.getId());
 		request.setVirtualSystemIds(vsIds);
 		Mockito.when(this.service.validator.validateAndLoad(request)).thenReturn(this.sfc);
 		// Act.
+		//update sfc with virtual system ids list {2,1}
 		BaseJobResponse response = this.service.dispatch(request);
+		
+		//check Virtual system is as updated
+		List<Long> sfcVsIdList = this.em.find(ServiceFunctionChain.class, this.sfc.getId()).
+				getVirtualSystems().stream().map(vss -> vss.getId()).collect(Collectors.toList());
+		//read back sfc and cross check with local Virtual system list for order
+		Assert.assertEquals(vsIds, sfcVsIdList);
 		Assert.assertNotNull("The returned response should not be null.", response);
 	}
 	
-	//TODO : karimull  Add more test cases in looking for updated in order. Need some ramp up here
+	@Test
+	public void testDispatch_sfcVirutalSystemIdsNotInOrder_ValidationFails() throws Exception {
```

Comment:
This name does not align with the test code. The validator is not mocked to fail and the sfc seems to be updated. So in essence this tests looks like the previous one.

Reply:
Agreed , it is kind of similar to previous one, but thought of checking in reverse also. 
May not needed ..taking it off.